src/components/NavContainer.tsx

```diff
@@ -0,0 +1,47 @@
+import * as React from 'react';
+import { Text, View } from 'react-native';
+import { NavigationContainer } from '@react-navigation/native';
+import { createBottomTabNavigator } from '@react-navigation/bottom-tabs';
+import Ionicons from 'react-native-vector-icons/Ionicons';
+
+import HomeScreen from '../screens/HomeScreen';
```

Comment:
I think it's better to have an index file in src/screens that exports all screens in src/screens. Then, we can do something like `import { HomeScreen, TaskScreen, ...} from '../screens';` so we don't need multiple separate import statements like this.

Reply:
^ this type of file is called a [barrel file](https://basarat.gitbook.io/typescript/main-1/barrel) (the link is in TS but works for any language), which like Thai mentions is useful for keeping imports clean and at a higher abstraction

Reply:
Oh wow I use it so often and I didn't even know what it was actually called lol, thanks Shravan

Reply:
that's lit. i see u.

Reply:
let's go with the barrel file idea.

Reply:
This is a great idea, thanks for sharing the link. Learned something new :) I will add this to the PR.